### PR TITLE
Add source timeout configuration column

### PR DESCRIPTION
**Pull request by octavia-squidington-iii**
## What
Create a Migration to create the new column that will allow to configure the source heartbeat timeout.

Tested localy.

source: https://github.com/airbytehq/airbyte/pull/22678

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
## Platform Test Results
   242 files  +   242     242 suites  +242   33m 6s [:stopwatch:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "duration of all tests") + 33m 6s
1 670 tests +1 670  1 659 [:heavy_check_mark:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "passed tests") +1 659  11 [:zzz:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "skipped / disabled tests") +11  0 [:x:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "failed tests") ±0 
1 689 runs  +1 689  1 678 [:heavy_check_mark:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "passed tests") +1 678  11 [:zzz:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "skipped / disabled tests") +11  0 [:x:](https://github.com/EnricoMi/publish-unit-test-result-action/blob/v1.20/README.md#the-symbols "failed tests") ±0 

Results for commit 4c6ba2c9. ± Comparison against base commit 3946c7ee.

[test-results]:data:application/gzip;base64,H4sIAPqK6mMC/03MQQ7CIBCF4as0rF04tMDgZcwUISG2xVBYGe8urbWw/L/JvDdzfrIru3V84JeOrdmnJh85UvJhKQAaZZFyTdsdpLr+875mY3YTutrTvzaDUxz5qUh9szGGeEjMy28W9VF1VeFJzegOzebe7aQJ8+xTCTYYORI3mpzoLe/BosUBR6FQkTAEoAFJcsc+XxCOKfcQAQAA

:recycle: This comment has been updated with latest results.

source: https://github.com/airbytehq/airbyte/pull/22678#issuecomment-1424900879

**Comment by octavia-squidington-iii** [comment]
### Airbyte Code Coverage
|File|Coverage [100%]|:green_apple:|
|:-|:-:|:-:|
|[V0_40_32_001__AddSourceHeartbeatConfiguration.java](https://github.com/airbytehq/airbyte/blob/cf5b97e74b7734eb2497f074bd030da062dbe7ba/airbyte-db%2Fdb-lib%2Fsrc%2Fmain%2Fjava%2Fio%2Fairbyte%2Fdb%2Finstance%2Fconfigs%2Fmigrations%2FV0_40_32_001__AddSourceHeartbeatConfiguration.java)|100%|:green_apple:|

|Total Project Coverage|24.67%|:x:|
|:-|:-:|:-:|

source: https://github.com/airbytehq/airbyte/pull/22678#issuecomment-1428464691

**Comment by benmoriceau** [comment]
This PR was already approved in the airbyte/airbyte repo

File: airbyte-db/db-lib/src/main/resources/configs_database/schema_dump.txt [hunk 1]
```diff
@@ -261,6 +262,7 @@ alter table "public"."workspace_service_account"
   add constraint "workspace_service_account_workspace_id_fkey"
     foreign key ("workspace_id")
     references "public"."workspace" ("id");
+comment on column "public"."actor_definition"."max_seconds_between_messages" is 'Define the number of seconds allowed between 2 messages emitted by the connector before ';
```
Comment (jdpgrailsdev) [inline airbyte-db/db-lib/src/main/resources/configs_database/schema_dump.txt:265]: Did the comment message get cut off?  Seems to be missing something after "before".

File: airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_40_32_001__AddSourceHeartbeatConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  private static void addSourceHeartbeatConfiguration(final DSLContext ctx) {
+    ctx.alterTable("actor_definition")
+        .addColumnIfNotExists(DSL.field(
+            "max_seconds_between_messages",
+            SQLDataType.INTEGER.nullable(true)))
+        .execute();
+
+    ctx.commentOnColumn(DSL.name("actor_definition", "max_seconds_between_messages"))
+        .is("Define the number of seconds allowed between 2 messages emitted by the connector before ")
```
Comment (jdpgrailsdev) [inline airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_40_32_001__AddSourceHeartbeatConfiguration.java:36]: Same comment:  is this cut off due to some limit or is it just missing the rest of the comment?

Comment (benmoriceau) [reply airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_40_32_001__AddSourceHeartbeatConfiguration.java:36]: No, it was a typo from previous PR..

**Review by jdpgrailsdev — APPROVED** [review]
:shipit:

**Review by gosusnp — APPROVED** [review]

**Comment by octavia-approvington** [comment]
What are we doing again? <br/> merge and squash

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**